### ISSUE TITLE
View not loaded and not firing State change handlers to update the view with new state

### ISSUE THREAD

**Issue by georgerajesh** [openanthem/nimbus-core#660]
# Issue Details

**Type of Issue**  (check one with "X")
```
[x] Bug Report => Please search GitHub for a similar issue or PR before submitting
[ ] Feature Request => Please ensure feature is not already in progress
[ ] Support Request => Please do not submit support requests here, instead see: https://discourse.oss.antheminc.com/
```

## Current Behavior:
View not updated through the State Change handlers are evaluated onload of view
Expected Behavior:
View should load and it should fire State change handlers and update the view with new state.
How to Reproduce the Issue:
Create a view root. Create a param at view root level with LabelConditional (as example) and set the state from Section Initialize. The LabelConditional has to be evaluated and view must be updated.
Steps to Reproduce:
Create a view root. Create a param at view root level with LabelConditional (as example) and set the state from Section Initialize. The LabelConditional has to be evaluated and view must be updated.
Code Snippet:
Environment Details
• Nimbus Version: 1.3.1.M4
• Browser: Chrome

**Comment by sandeepmantha** [comment]
Available in 1.3.1.M7

**Comment by dileeproopreddy** [comment]
Fix is working as expected
